src/perl/Permabit/KernelModule.pm

```diff
@@ -78,8 +79,15 @@ sub loadFromFiles {
   my ($self) = assertNumArgs(1, @_);
   my $modName = $self->{modName};
   my $modVer = $self->{modVersion};
-
-  my $loaded = $self->SUPER::loadFromFiles();
+  my $loaded = 0;
+
+  # With useUpstream test, we don't need to load upstream
+  # module(dm-vdo), since it is in the kernel already.
+  if ($self->{useUpstream}) {
+    return 1;
+  } else {
+    $loaded = $self->SUPER::loadFromFiles();
+  }
```

Comment:
The point of use return 1; here is that you don't need the "else" branch. $loaded can be defined just like it used to be.